Libraries/JavaScriptAppEngine/Initialization/loadSourceMap.js

```diff
@@ -17,8 +17,6 @@ var RCTSourceCode = require('NativeModules').SourceCode;
 var SourceMapConsumer = require('SourceMap').SourceMapConsumer;
 var SourceMapURL = require('./source-map-url');
 
-var fetch = require('fetch');
```

Comment:
can you make sure source maps are still working. trigger a redbox someway and make sure the file displayed is correct